src/jalapeno-pickled-eggs.md

```diff
@@ -0,0 +1,32 @@
+# Jalapeno Pickled Eggs
+
+![eggs fermenting in jar](pix/jalapeno-eggs.webp)
```

Comment:
```suggestion
![eggs fermenting in jar](pix/jalapeno-pickled-eggs.webp)
```

Best to make the image file name the same as the recipe it belongs with. I can't add a comment on the actual image file, but obviously making this change will mean renaming the image as well.